### PR TITLE
feat: [NFTG-9] Added Glassy Social Icons component

### PR DESCRIPTION
**Pull request by SiddheshShinde-tech**

### CODE REVIEW

File: src/components/nav/LandingNav.js [hunk 1]
```diff
@@ -13,12 +15,36 @@ function LandingNav() {
...
+              <ShineButton> Discover</ShineButton>
+            </Link>
           </ContentItem>
 
           <ContentItem showOnMedia={true}>
             <ShineButton>Enter App</ShineButton>{" "}
           </ContentItem>
+
+          <ContentItem>
+            <GlassyIconsStrip>
```
Comment (adeeshsharma) [inline src/components/nav/LandingNav.js:29]: Directly use these in the Top Nav

File: src/pages/Landing/index.js [hunk 2]
```diff
@@ -42,6 +52,39 @@ function Landing() {
...
+        />
+        <GlassyIcons
+          routepath="/test"
+          color="#fff"
+          icon={faInstagram}
+          iconSize="xs"
+          height="40px"
+          width="40px"
+        />
+      </GlassyIconsStrip>
```
Comment (adeeshsharma) [inline src/pages/Landing/index.js:87]: looks great!
Can be removed from here, Place these in the stonk society social strip component instead 👍🏻

Comment (SiddheshShinde-tech) [reply src/pages/Landing/index.js]: Resolved in the latest commit.

File: src/components/nav/LandingNav.js (hunk 1 above)
Comment (SiddheshShinde-tech) [reply src/components/nav/LandingNav.js]: Resolved in the latest commit

**Pull request merged**